README.md

```diff
@@ -117,6 +117,18 @@ Only works with version '2' docker-compose.yml configuration files
 
 This option can also be configured on the agent machine using the environment variable `BUILDKITE_PLUGIN_DOCKER_COMPOSE_IMAGE_REPOSITORY`.
 
+## `image-tag` (optional)
+
+If you want to use a custom tag, please set this option.
+
+The default is `"$project_name-$COMPOSE_SERVICE_NAME-build-$BUILDKITE_BUILD_NUMBER"`.
```

Comment:
We could probably support interpolation with [`envsubst`](https://www.gnu.org/software/gettext/manual/html_node/envsubst-Invocation.html).